src/action-ros-ci.ts

```diff
@@ -201,7 +190,7 @@ async function run() {
 		for (let vcsRepoFileUrl of vcsRepoFileUrlListResolved) {
 			await execBashCommand(
 				`curl ${curlFlags} '${vcsRepoFileUrl}' | vcs import --force --recursive src/`,
-				commandPrefix,
+				"",
```

Comment:
Could you s/""/undefined/ as it's an optional argument? (everywhere)

See https://stackoverflow.com/questions/30734509/how-to-pass-optional-parameters-while-omitting-some-other-optional-parameters